src/test/groovy/com/benjaminsproule/swagger/gradleplugin/MissingImplicitParamsITest.groovy

```diff
@@ -30,7 +30,7 @@ class MissingImplicitParamsITest extends AbstractPluginITest {
 
         then:
         result.task(":${GenerateSwaggerDocsTask.TASK_NAME}").outcome == FAILED
-        result.output.contains("Caused by: java.lang.ClassNotFoundException: com.benjaminsproule.swagger.gradleplugin.test.model.MissingRequestModel")
+        result.output.contains("Unrecognized Type: [null]")
```

Comment:
This is a completely different and less helpful error message. It would be nice if instead of null it could print out the expected type.

Reply:
This is the result of using io.swagger.util.ReflectionUtils#typeFromString. If you can tell me where it comes from - I can try and fix it.